requirements-base.txt

```diff
@@ -68,7 +68,7 @@ urllib3[brotli]>=1.26.9
 brotli>=1.0.9
 # See if we can remove LDFLAGS from lib.sh
 # https://github.com/getsentry/sentry/pull/30094
```

Comment:
we're very close to being able to do this!  though pyuwsgi doesn't have an M1 wheel on pypi so we'll still need this temporarily for now